src/bundle/Resources/views/Security/login.html.twig

```diff
@@ -4,7 +4,7 @@
     <form action="{{ path( 'login_check' ) }}" method="post" role="form">
         <fieldset>
             {% if error %}
-                <span>{{ error.message }}</span>
+                <span>{{ error.message|trans }}</span>
```

Comment:
Thank you for contribution. Can you add a description filter as well? Something like `error.message|transdesc('Cannot login because of error.')` or similarly.

Reply:
I think that would change whole behavior of this. That's like really bad solution.

Reply:
hi @ViniTou , you mean bad solution the error.message|trans part or the desc part? thanks!.